packages/main/src/themes/RadioButton.css

```diff
@@ -42,44 +41,29 @@
 }
 
 /* focused */
-:host(:not([disabled])) .ui5-radio-root:focus:before {
+:host(:not([disabled])) .ui5-radio-root:focus::before {
 	content: "";
 	display: var(--_ui5_radio_button_focus_outline);
 	position: absolute;
 	top: var(--_ui5_radio_button_focus_dist);
 	bottom: var(--_ui5_radio_button_focus_dist);
-	left: var(--_ui5_radio_button_focus_dist);
-	right: var(--_ui5_radio_button_focus_dist);
+	inset-inline-start: var(--_ui5_radio_button_focus_dist);
+	inset-inline-end: var(--_ui5_radio_button_focus_dist);
```

Comment:
all these can be rewritten with just `inset` since the distance is the same on every side

Reply:
Done